network/eni/eni_test.go

```diff
@@ -0,0 +1,33 @@
+package eni
```

Comment:
Should we add the copyright header to the top of the file?

Reply:
good catch, added